src/library/module.cpp

```diff
@@ -521,13 +521,13 @@ environment add_inductive(environment                       env,
                           inductive::inductive_decl const & decl,
                           bool                              is_trusted) {
     pair<environment, certified_inductive_decl> r = inductive::add_inductive(env, decl, is_trusted);
-    environment new_env            = r.first;
-    certified_inductive_decl cdecl = r.second;
+    environment new_env             = r.first;
+    certified_inductive_decl cidecl = r.second;
     module_ext ext = get_extension(env);
```

Comment:
I imagine `cdecl` is a reserved keyword on VS, right? :)

Reply:
yep.